src/pgduckdb_hooks.cpp

```diff
@@ -51,10 +52,10 @@ ContainsCatalogTable(List *rtes) {
 		}
 
 		if (rte->relid) {
-			auto rel = RelationIdGetRelation(rte->relid);
-			auto namespace_oid = RelationGetNamespace(rel);
+			Relation rel = RelationIdGetRelation(rte->relid);
```

Comment:
Just curious - why the change/explicit type specification?
Should we prefer it over `auto`?

Reply:
No good reason, mainly because the type isn't that long to type and it gives a bit more info when looking at the code on github.